demo/Cargo.toml

```diff
@@ -61,13 +59,15 @@ bevy_tasks = "0.5.0"
 default = [
     "sdl2-bundled",
     "sdl2-static-link",
-    "profile-with-puffin",
-    #"profile-with-stats-alloc", # Cannot be enabled with "profile-with-tracy-memory". Will run renderer / game loop single-threaded.
+    #"profile-with-puffin",
```

Comment:
Is it necessary for `profile-with-puffin` to be removed from the default options?

Reply:
No, good point. :)